cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandConfiguration.java

```diff
@@ -12,6 +12,12 @@ public interface DebeziumOperandConfiguration {
     @WithDefault(DebeziumConstants.DEFAULT_IMAGE_PULL_SECRET_NAME)
     LocalObjectReference imagePullSecretsName();
 
+    @WithDefault(DebeziumConstants.DEFAULT_APICURIO_AUTH_SERVICE_URL)
+    String apicurioAuthServiceUrl();
```

Comment:
If we already know the property values that are going to be used in Stage/Prod, and if they are different from the defaults, it might be nice to include them here: https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/blob/main/etc/kubernetes/manifests/overlays/staging/cos-fleetshard-operator-debezium/app-config/application.properties

Reply:
Good catch! @ani-sha could you please add the config for stage?
Not sure, but I think the URL for stage is `https://identity.api.stage.openshift.com/auth`.
And I think the realm for prod and stage will be `rhoas-kafka-sre`, or will it stay as `rhoas`? Do you know that @rinaldodev?